apps/mivisionx_openvx_classifier/README.md

```diff
@@ -55,35 +58,39 @@ Usage: ./classifier
 
 #### Generating weights.bin for different Models
 
-1. Download or train your own caffemodel for the supported models listed above.
+1. Download or train your caffemodel for the supported models listed above.
 
 Here is the sample download [link](https://github.com/SnailTyan/caffe-model-zoo) that contains all the prototxt: 
 
 2. Use [MIVisionX Model Compiler](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/tree/master/model_compiler#neural-net-model-compiler--optimizer) to extract weights.bin from the pre-trained caffe models
+
 **Note:** MIVisionX installs all the model compiler scripts in `/opt/rocm/mivisionx/model_compiler/python/` folder
 
+ + Convert the pre-trained caffemodel into AMD NNIR model:
+
+  ``` 
```

Comment:
Codacy found an issue: [[list-item-content-indent] Don’t use mixed indentation for children, add 2 spaces](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6056700)